### PR TITLE
Add ZumoSwap Application

### PR DESCRIPTION
**Pull request by Tinto1024**
# Grant Application Checklist

- [x] The [`applications/application-template.md`](../applications/application-template.md) or [`applications/research-template.md`](../applications/research-template.md) has been copied, renamed ( "project_name.md") and updated.
- [x] An Ethereum address for the payment of the milestones is provided inside the application.  
- [x] The total funding amount of the project is up to USD $30k for every grant applications.
- [x] The initial PR contains only one commit (squash if needed before submitting your PR).
- [x] If you own an Economic X Node, make sure you have already sent an email([`applications/email-template.md`](../applications/email-template.md)) to grant@vechain.org to claim the additional grant.

### CODE REVIEW

File: applications/zumoswap.md [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+
+At this stage, we built on top of VeChain Foundation open-source DEX code. We hired devs and designers, the work has been done is summarized as the following:
+1) Add any-to-any token swap (route finding, rate calcuate).
+2) Add any-to-any pool liquidity add/remove interface.
+3) Improve the query of users' tokens balances and pool status to multi-clause calls.
+4) Improve swap and liquidity-related actions to a multi-clause transaction.
+
+#### M1.1 Launch: Server costs for 12 months
+
+We have rented servers and run a VeChain node for data scrapping. We hope the foundation can help us to cover the intial costs, but since 13th month we will earn money and sponsor ourselves.
```
Comment (57u) [inline applications/zumoswap.md:90]: typo: data scraping

File: applications/zumoswap.md [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+
+At this stage, we will contact the foundation to sponsor VTHO to cover our bootstrap trading competition for the VeUSD launching event.
+
+The money will be solely used for paying users' transactions to boost on-chain transactions.
+
+#### M3. Charts
+
+A chart similar to (https://info.uniswap.org/#/) is a key part of the DEX info. This requires a database, a set of APIs and a web page to display. We expect this to be a full-time engineer working for a full month and a designer for two weeks.
+
+The working result will be a web page and a set of **open-source Javascript or Python SDKs** for scrapping data. This can largely benefit the VeChain Community.
```
Comment (57u) [inline applications/zumoswap.md:122]: typo: ...for scraping data.

**Review by 57u — CHANGES_REQUESTED** [review]

**Comment by NecoSherry** [comment]
@Tinto1024 Thank you for your application! Sounds great. We will look into it as soon as possible. 

I had a quick look, and I had a few comments:
1. I noticed that your team members required anonymity. In my option it would be helpful if the evaluators and the committees learnt more about your team. Would it be all right if you sent the team information privately to grant@vechain.org? 

2. Could you format your milestones according to [the application template](https://github.com/vechain/grant-program/blob/master/applications/application-template.md)? 
   - Update the overview table with total funds
   - Update the overview table with full-time equivalent of each milestone. 
   - Table the deliverable of each milestone

3. Would you please update more details about the future operation plan if you have one?

**Comment by Tinto1024** [comment]
> @Tinto1024 Thank you for your application! Sounds great. We will look into it as soon as possible.
> 
> I had a quick look, and I had a few comments:
> 
> 1. I noticed that your team members required anonymity. In my option it would be helpful if the evaluators and the committees learnt more about your team. Would it be all right if you sent the team information privately to [grant@vechain.org](mailto:grant@vechain.org)?
> 2. Could you format your milestones according to [the application template](https://github.com/vechain/grant-program/blob/master/applications/application-template.md)?
>    
>    * Update the overview table with total funds
>    * Update the overview table with full-time equivalent of each milestone.
>    * Table the deliverable of each milestone
> 3. Would you please update more details about the future operation plan if you have one?

Total funds: Fixed.
full-time equivalent: Fixed.
Table the deliverable of each milestone: It is already included in the `Details` section.

**Pull request closed without merging**